src/cypress/integration/pages/TradePage.js

```diff
@@ -0,0 +1,79 @@
+class TradePage {
+  getSymbol(symbol) {
+    return cy.get('div[data-testid="tile-' + symbol.replace('/', '') + '"]')
```

Comment:
`symbol.replace('/', '')` is used a lot throughout, consider creating a util function for it to reduce duplication.

Reply:
Also, would `getTile` be a more appropriate name?